addons/rating/migrations/10.0.1.0/openupgrade_analysis_work.txt

```diff
@@ -0,0 +1,7 @@
+---Fields in module 'rating'---
+rating       / rating.rating            / consumed (boolean)            : NEW
```

Comment:
This analysis file is not correct... this field exists also in v10, but other new fields like `rating_image` or `rating_text` are not reported. @StefanRijnhart do you have idea why?

Reply:
Ah @pedrobaeza you are looking at the analysis of 10.0 so this field should be here. As for rating_image and rating_text, these are computed fields and have been traditionally left out of the analysis as they presumably do not need any work.

Reply:
Agh, I have mixed v10 and v11 as we have both. Sorry for the noise.